client/app/index.html

```diff
@@ -70,6 +70,7 @@
     <script src="scripts/superdesk-authoring/widgets/widgets.js"></script>
     <script src="scripts/superdesk-authoring/comments/comments.js"></script>
     <script src="scripts/superdesk-authoring/versioning/versions.js"></script>
+    <script src="scripts/superdesk-authoring/history/history.js"></script>
```

Comment:
For unittests you should add to test-main.js the following line:
tests.push('superdesk-authoring/history/history');